packages/kbn-gateway/index.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { bootstrap } from './src';
+
+bootstrap();
```

Comment:
Agreed, I kept it this way because it's easier for development purposes, and I was thinking this is something we'd do as a part of https://github.com/elastic/kibana/issues/141886. But I'm now thinking about addressing that in this PR as well so we don't have something merged that's going to change a bunch later